test/unit/math/rev/core/vari_test.cpp

```diff
@@ -8,3 +8,10 @@ TEST(AgradRev, insertion_operator) {
   ss << &v;
   EXPECT_EQ("5:0", ss.str());
 }
+
+TEST(AgradRev, int_test) {
+  stan::math::vari_value<int> v(5);
+  std::stringstream ss;
+  ss << &v;
```

Comment:
This is printing a `stan::math::vari_value<int>*` type?

Why isn't it just `ss << v;`?

Reply:
Oh I see the implementation. Makes sense.